.github/workflows/publish-shared-sdk.yml

```
@@ -48,25 +48,25 @@ jobs:
           ./scripts/gen-go-shared-sdk.sh
           rm -rf $GITHUB_WORKSPACE/hcp-sdk-go/temp
 
+      - name: Create branch with updated service SDK
+        run: |
+          cd $GITHUB_WORKSPACE/hcp-sdk-go
+          git config user.name "HashiCorp Cloud Services"
+          git config user.email "59096967+hashicorp-cloud@users.noreply.github.com"
+          service_branch_exists="$(git ls-remote --heads origin auto-update-cloud-shared-sdk)"
+          [[ -n $service_branch_exists ]] && git push origin --delete auto-update-cloud-shared-sdk
+          git checkout -b auto-update-cloud-shared-sdk
+          git add clients/cloud-shared/*
+          git commit -m "Update Cloud API SDK"
```

To match the generated client name
```suggestion
          git commit -m "Update cloud-shared SDK"
```